doc/index.rst

```diff
@@ -1,8 +1,31 @@
 IPv8 Documentation
 ==================
 
-Welcome to the IPv8 documentation!
-This file will help you navigate the existing documentation.
+Welcome to the IPv8 documentation! This file will help you navigate the existing documentation.
+The documentation will explain the concepts behind IPv8, the architecture, and guide you to making your first overlay.
+
+IPv8 Features
+==================
+
+The IPv8 overlay offers network connection to a known digital identity which is robust against several network problems and security issues. When your external IPv4 network address changes, IPv8 keeps the network connectivity going. IPv8 offers communication based on public keys, it uses a custom NAT-traversing DHT to find their current IPv4 network address. The overlay keeps track of a number of neighbors and occasionally checks if they are still responsive. IPv8 is a networking layer which offers identities, communication with some robustness, and provides hooks for higher layers. For instance, our Internet-deployed reputation functions and ledger-based storage of reputation data. IPv8 in combination with our Trustchain is designed as a mechanism to build trust. 
+
+It offers public-key based identities and offers protection against man-in-the-middle attacks, some network failures, change of IP numbers, network mobility, and blocking by NAT/Firewalls. IPv8 offers global connectivity through integrated UDP NAT puncturing, announcement of your identity claim and a web-of-trust. IPv8 has an integrated attestation service. You can use IPv8 for official verification that something is true or authentic, according to a trustworthy attestor. By using zero-knowledge proofs we attempt to minimize privacy leakage.
```

Comment:
This seems mostly like a repeat of the last paragraph. Only the last 3 sentences offer something new: strong identity.